index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="fr">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>HACKTOBERFEST-GIT-COURSE</title>
+</head>
+<body>
+    <h1>HACKTOBERFEST-GIT-COURSE</h1>
+    <p>Page d'accueil du cours sur git</p>
+</body>
+</html>
```

Comment:
Do you see the forbidden mark at the end of the last line? It indicates a new-line character is missing.

Could you please add a new-line character at the end of the last line in the file?